NEWS.md

```diff
@@ -41,6 +41,8 @@ plumber 0.5.0
 
 ### Minor new features and improvements
 
+* The `curl` CLI tool is now installed in the `trestletech/plumber` docker image. ([#457](https://github.com/trestletech/plumber/issues/457))
```

Comment:
Can you add your name next to the issue? (Similar to `@byzeng` below.)

Reply:
Done in 7366b11